### PR TITLE
Biogenerator update

### PR DESCRIPTION
**Pull request by Minelol76**
Biogenerators now can create sheets of paper for 10 biomass per sheet.

## About The Pull Request

The biogenerator can now produce new item: sheet of paper for 10 biomass.

## Why It's Good For The Game

This minor change lets botany help all departments with paperwork, especially to cargo since paper can only be found and bought in supply console. It also makes sense for biogenerator to create paper since paper is generally made out of wood.

## Changelog
:cl:
add: Added ability for Biogenerators to create paper.

/:cl:

### CODE REVIEW

File: code/modules/research/designs/biogenerator_designs.dm [hunk 1]
```diff
@@ -194,3 +194,11 @@
 	materials = list(/datum/material/biomass= 300)
 	build_path = /obj/item/clothing/head/rice_hat
 	category = list("initial","Organic Materials")
+
+/datum/design/paper
+	name = "Sheet of Paper"
+	id = "paper"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass= 10)
+	build_path = /obj/item/stack/sheet/paper
```
Comment (Firefox13) [inline code/modules/research/designs/biogenerator_designs.dm:203]: It should be(I think) /obj/item/paper
I cannot suggest a change, so make it yourself.
Where did you got this path anyway?

Comment (yorii) [reply code/modules/research/designs/biogenerator_designs.dm:203]: I think they just copied an earlier path and replaced leather or something with paper, but paper has a completely different object path :p

**Comment by ToasterBan** [comment]
on one hand this sounds cool, on the other when is there EVER a paper shortage?

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
/obj/item/stack/sheet/paper Does not exist

**Pull request closed without merging**